src/file-handlers.ts

```diff
@@ -268,13 +283,28 @@ function renderFile(
  * ```
  */
 
+function renderFileAsync(
+  filename: string,
+  data: DataObj,
+  config?: PartialConfig
+): Promise<string>
+
+function renderFileAsync(
+  filename: string,
+  data: DataObj,
+  config: PartialConfig,
+  cb: CallbackFn
+): void
+
 function renderFileAsync(
   filename: string,
   data: DataObj,
   config?: PartialConfig,
   cb?: CallbackFn
 ): Promise<string> | void
 
+function renderFileAsync(filename: string, data: DataObj, cb: CallbackFn): void
+
 function renderFileAsync(filename: string, data: DataObj, cb?: CallbackFn): Promise<string> | void
```

Comment:
Could we remove this line?

Reply:
yes, should be resolved, see https://github.com/eta-dev/eta/pull/199/commits/fcc1845dc1da2d105283e50aa44f60350ec010d6